cypress/src/support/component-index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
```

Comment:
What purpose does this file serve? Can it be removed?

Reply:
Yes. It was there already when I cloned it so thought it is part of your cypress code but if not I will remove it.